tests/performance_testing/scenario-creation/config.py

```diff
@@ -11,8 +11,8 @@
 
 import datetime as dt
 
+import modin.pandas as pd
```

Comment:
I believe we should keep pandas as the default usage in this config.py file.

Other wise we should update the config to have both in parallel modin and pandas